sharktank/sharktank/models/llama/testing.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2024 Advanced Micro Devices, Inc
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from typing import List
+
+import torch
+
+from ...types.tensors import *
+from ...types.theta import Theta
+
+
+def make_rand_torch(shape, dtype):
```

Comment:
Add small comment that you are shifting the distribution

Reply:
Good suggestion, have done